.gitignore

```diff
@@ -8,6 +8,7 @@ workers/unity/build/
 .idea/
 .vs/
 .DS_Store
+*.orig
```

Comment:
What's this?

Reply:
git creates those during rebase/merge. https://stackoverflow.com/questions/1251681/git-mergetool-generates-unwanted-orig-files

I'll configure my git to not keep a backup, but others might not do the same, so this stops them from being committed accidentally.